### ISSUE TITLE
Hello, This is a chinese voyager document,I hope I can help us

### ISSUE THREAD

**Issue by tu6ge** [thedevdojo/voyager#4722]
[chinese document](http://doc.laravel-voyager.cn/)

Thanks

**Comment by MrCrayon** [comment]
I don't think it would be easy to mantain translations in the official site https://voyager-docs.devdojo.com/ but this could still be helpful.

xiexie
